scripts/install-aws.sh

```diff
@@ -23,22 +24,24 @@ main () {
   cp /app/aws-cni-support.sh /host/opt/cni/bin/
   cp /app/portmap /host/opt/cni/bin/
 
-  echo "===== Starting amazon-k8s-agent ==========="
+  echo "====== Starting amazon-k8s-agent ======"
   /app/aws-k8s-agent &
 
   # Check ipamd health
   echo "Checking if ipamd is serving"
   waitIPamDServing
 
-  echo "===== Copying AWS CNI plugin and config ========="
+  echo "Copying AWS CNI plugin and config"
```

Comment:
nit: Break this down into two steps
* Moving the cni-plugin
* Moving the cni-conf

Reply:
Will fix in master branch.